### ISSUE TITLE
UI changes for "Remove all remote watches"

### ISSUE THREAD

**Issue by djwhatle** [konveyor/mig-ui#1003]
@ibolton336 @vconzola after our conversation, will keep this GH issue updated with latest known UI requirements to support https://github.com/konveyor/mig-controller/pull/651

---

1.  **(Non display layer)** Need to set `migplan.spec.refresh=true` when re-entering the "edit" migplan wizard for an existing plan. Wait on `migplan.spec.refresh=false` and `migplan.status.conditions[RefreshInProgress].IsAbsent()` to indicate PV discovery is done before showing user PV migration options.

---

2. ~~**(Non display layer)** Ensure that _both_ launching MigStorage edit modal _and_ clicking "Check Connection" buttons for MigStorage will set `migstorage.spec.refresh: true`. Use `observedDigest` to wait for completion.~~

We do MigStorage connection checking every 30s in mig-controller. Not needed.

---

3. ~~**(Non display layer)** Ensure that _both_ launching MigCluster edit modal _and_ clicking "Check Connection" buttons for MigCluster will set `migcluster.spec.refresh: true`. Use `observedDigest` to wait for completion.~~

We do MigStorage connection checking every 30s in mig-controller. Not needed.

---

4. Moved to https://github.com/konveyor/mig-ui/issues/1012

**Comment by djwhatle** [comment]
@ibolton336 in case you're curious, the full list of possible conditions on the plan are here:
https://github.com/konveyor/mig-controller/blob/master/pkg/controller/migplan/validation.go#L29

**Comment by vconzola** [comment]
@djwhatle @ibolton336 I'm wondering if #4 should be a separate issue since displaying detailed info about the reason a plan isn't ready is something we should do irrespective of removing remote watches #651?

**Comment by djwhatle** [comment]
@vconzola that's a good point, I'll break it out to a separate issue.
